src/sonic-config-engine/minigraph.py

```diff
@@ -643,6 +643,52 @@ def parse_xml(filename, platform=None, port_config_file=None):
             count += 1
         results['MIRROR_SESSION'] = mirror_sessions
 
+ 
+    spine_chassis_frontend_role = 'SpineChassisFrontendRouter'
+    chassis_backend_role = 'ChassisBackendRouter'
+
+    # Special parsing for spine chassis frontend routers
+    if current_device['type'] == spine_chassis_frontend_role:
+        chassis_vnet ='Vnet1'
+        chassis_vxlan_tunnel = 'tunnel1'
+        chassis_vni = 8000
```

Comment:
I think we should not hard code here. We can hardcode in the template.

Reply:
This information (Vnet name, Vxlan tunnel name and vni) will be inserted to config DB. I don't know which template you refer to.

Reply:
bgpd.j2

Reply:
@lguohan I think this information is also necessary for fpmsyncd and vxlanmgrd